### PR TITLE
Notify Quantum Metric of user AB test participations 

### PR DESCRIPTION
**Pull request by GHaberis**
## What are you doing in this PR?

We need to notify Quantum Metric of a user's AB test participations, using the `window.QuantumMetricAPI.sendEvent` function. This PR sends this relevant data to Quantum Metric.

Whilst doing this I've also updated the logic to load Quantum Metric's script by replacing some custom logic to add the script with `import { loadScript } from '@guardian/libs'`. 

[**Trello Card**](https://trello.com/c/BfI7Ru7b/462-quantum-metric-implement-sendevent-call-for-ab-test-participations)

### CODE REVIEW

**Comment by GHaberis** [comment]
[Quantum Metric: Implement SendEvent call for AB test participations.](https://trello.com/c/BfI7Ru7b/462-quantum-metric-implement-sendevent-call-for-ab-test-participations)

File: support-frontend/assets/helpers/tracking/quantumMetric.ts [hunk 1]
```diff
@@ -1,46 +1,87 @@
...
+				crossOrigin: 'anonymous',
+			},
+		)
+			.then(() => {
+				resolve();
+			})
+			.catch(() => {
+				logException('Failed to load Quantum Metric');
+			});
+	});
```
Comment (i-hardy) [inline support-frontend/assets/helpers/tracking/quantumMetric.ts:75]:
```suggestion
	return loadScript(
			'https://cdn.quantummetric.com/instrumentation/1.31.3/quantum-gnm.js',
			{
				async: true,
				integrity:
					'sha384-jL1rlM/U30WWRsDEQDalNiDV8FOjayD5RCgzkywrohMqg6oME3zbszWB31/40MAD',
				crossOrigin: 'anonymous',
			},
		)
			.catch(() => {
				logException('Failed to load Quantum Metric');
			});
	});
```

If `loadScript` returns a promise there's no need to wrap it in the promise constructor.

**Review by i-hardy — APPROVED** [review]
One suggestion, otherwise looks good!

File: support-frontend/assets/helpers/tracking/quantumMetric.ts (hunk 1 above)
Comment (GHaberis) [reply support-frontend/assets/helpers/tracking/quantumMetric.ts:75]: It does indeed return a Promise, good suggestion 👍

**Pull request merged**